### PR TITLE
Adds documentation for application overrides

### PR DESCRIPTION
**Pull request by nfmmonteiro**
Jira ticket: https://underwriteme.atlassian.net/browse/DAISY-3666

### CODE REVIEW

File: content/application/new.md [hunk 1]
```diff
@@ -47,6 +47,16 @@ Creating new Application based on sent Customers and Products JSON data.
 - _**owner**_ `string` *(optional)* - Owner of the application. If not specified, the authenticated user is used.
 - _**quoteEffectiveDate**_ `string` *(optional)* - The quote effective date. ISO 8601 date format (YYYY-MM-DD) required. Example: 1980-01-01.
 - _**originatorId**_ `string` *(optional)* - The id of the portal the application originated from.
+- _**overrides**_ `object` *(optional)* - Placeholder to specify overrides for the application.
```
Comment (laurenceweil) [inline content/application/new.md]: Can this be a bit more descriptive, why / when would this ever be used?

File: content/application/new.md [hunk 2]
```diff
@@ -47,6 +47,16 @@ Creating new Application based on sent Customers and Products JSON data.
...
+    - _**user**_ `object` *(optional)* - Placeholder to specify overrides for the user creating the application.
+        - _**name**_ `string` *(optional)* - The name of the agent (first name and last name).
+        - _**firm**_ `object` *(optional)* - Placeholder to specify overrides for the firm.
+            - _**name**_ `string` *(optional)* - The name of the firm
+            - _**frn**_ `string` *(optional)* - The firm reference number
+        - _**providerAgentDetails**_ `array` *(optional)* - List of agent number overrides. It allows to specify an agent number per provider and commission style (represented as `object`).
```
Comment (laurenceweil) [inline content/application/new.md]: Probly slight reword to "It allows a user to specify an agent number per provider and commission style (represented as `object`)."

File: content/application/new.md [hunk 3]
```diff
@@ -47,6 +47,16 @@ Creating new Application based on sent Customers and Products JSON data.
...
+            - _**commissionStyle**_ `string` *(required)* - The commission style. Possible values are `INDEMNITY` or `NON_INDEMNITY`.
+            - _**providerId**_ `string` *(required)* - The ID of the provider, e.g. `TRUE_POTENTIAL`.
+            - _**agentNumber**_ `string` *(required)* - The agent number.
 
 
 ### Create new Application [POST]
```
Comment (laurenceweil) [inline content/application/new.md]: Can you add an example with overrides completed.

**Comment by lorinckemeny** [comment]
Can you please run the make command after your changes and add the updated apiary file to the PR, please?

**Comment by nfmmonteiro** [comment]
@laurenceweil @laurenceweil comments addressed, back to you.

**Pull request merged**